app/src/main/java/com/gmail/mxwild/newhabit/moviedetail/MovieDetailsViewModelFactory.kt

```diff
@@ -0,0 +1,13 @@
+package com.gmail.mxwild.newhabit.moviedetail
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+
+class MovieDetailsViewModelFactory() : ViewModelProvider.Factory {
```

Comment:
You don't need this factory. View provider will just use reflection to create model that has default constructor.

Reply:
Ok. Delete